.editorconfig

```diff
@@ -260,6 +260,9 @@ dotnet_diagnostic.IDE0059.severity = warning
 # IDE0060: Remove unused parameter
 dotnet_diagnostic.IDE0060.severity = warning
 
+# CA1012: Abstract types should not have constructors
+dotnet_diagnostic.CA1012.severity = warning
```

Comment:
❔ Can we move this to one of the **.globalconfig** files? The parsing performance of those files is better than **.editorconfig**, and they are more consistently applied to the code base.

Reply:
If we feel parsing performance is an issue, can we either file compiler bug for it to improve editorconfig performance? Also we should consider moving everything over from editorconfig to globalconfig instead of just one off changes.

Reply:
> If we feel parsing performance is an issue, can we file a compiler bug for it to improve editorconfig performance?

The **.editorconfig** parsing performance is very good, but by definition has to be applied per-file. **.globalconfig** doesn't need to consider paths. The latter is also preferable because duplication definitions trigger a warning rather than silently ignoring one of them.

Reply:
@sharwell Can this be tracked separately for everything in editorconfig as suggested by @mavasani?